include/sys/dsl_scan.h

```diff
@@ -153,6 +169,9 @@ typedef struct dsl_scan {
 	dsl_scan_phys_t scn_phys_cached;
 	avl_tree_t scn_queue;		/* queue of datasets to scan */
 	uint64_t scn_bytes_pending;	/* outstanding data to issue */
+
+	/* members needed for syncing rescrub status to disk */
+	dsl_rescrub_phys_t rescrub_phys; /* on disk representation of rescrub */
```

Comment:
This needs to start with `scn_`

Reply:
We have `dsl_scan_phys_t scn_phys` for scanning so I thought of doing it the same.   `dsl_rescrub_phys_t  rescrub_phys` for rescrubbing.